news/index.md

```diff
@@ -12,5 +12,8 @@ If you think you have found a security bug, or want to look at all the
 vulnerabilities we have published and fixed, visit the
 [Vulnerabilities](vulnerabilities.html) page.
 
+For a list of CVEs and their impact on the FIPS provider, visit the
+[CVEs and FIPS](fips-cve.html) page.
+
```

Comment:
Could you also link it from the Downloads page https://www.openssl.org/source/ ? There is a lot of text about FIPS provider in there so it seems good place to add this link as well.

Reply:
Added the link and reformatted the list of validated versions into a table.